### DISCUSSION TITLE
rapids gpu model can't use auto_stack

### DISCUSSION THREAD

**Discussion by leoin841023** [autogluon/autogluon#1845] [Q&A]
predictor = TabularPredictor(
    label='LOG_TOXICITY_PPM',
    eval_metric='pearsonr'
).fit(
    train,
    auto_stack='True',
    hyperparameters={
        
        KNNRapidsModel: {},
        LinearRapidsModel: {},
        RFRapidsModel: {},

    },
)

I got this error 
ValueError: The configured object store size (51.539607552 GB) exceeds /dev/shm size (51.539607552 GB). This will harm performance. Consider deleting files in /dev/shm or increasing its size with --shm-size in Docker. To ignore this warning, set RAY_OBJECT_STORE_ALLOW_SLOW_STORAGE=1.
No base models to train on, skipping auxiliary stack level 2...
No base models to train on, skipping stack level 2...
No base models to train on, skipping auxiliary stack level 3...

**Comment by Innixma** [comment]
Can you try without `auto_stack=True`? Also, providing full logs will be more useful than just a snippet.

**Reply by leoin841023** [reply]
If without auto_stack=True or preset='best_quality' autogluon can work


The error is below:

```
"  0%|          | 0/48 [00:00<?, ?it/s]2022-06-20 17:16:13,853\tERROR worker.py:79 -- Unhandled error (suppress with RAY_IGNORE_UNHANDLED_ERRORS=1): The worker died unexpectedly while executing this task. Check python-core-worker-*.log files for more information.\n",
      "2022-06-20 17:16:14,842\tERROR worker.py:79 -- Unhandled error (suppress with RAY_IGNORE_UNHANDLED_ERRORS=1): The worker died unexpectedly while executing this task. Check python-core-worker-*.log files for more information.\n",
"\tWarning: Exception caused NeuralNetMXNet_BAG_L1 to fail during training (ImportError)... Skipping this model.\n",
      "\t\t\u001b[36mray::_ray_fit()\u001b[39m (pid=873567, ip=10.64.19.132)\n",
      "ModuleNotFoundError: No module named 'mxnet'\n",
   "2022-06-20 17:16:31,257\tWARNING worker.py:1228 -- 3 retries left for task 47d445c93c6a7903ffffffffffffffffffffffff01000000, attempting to resubmit.\n",
      "2022-06-20 17:16:31,279\tWARNING worker.py:1228 -- A worker died or was killed while executing a task by an unexpected system error. To troubleshoot the problem, check the logs for the dead worker. RayTask ID: 5d556e4e31ca0f07094c56b666788b208903ddd813a8786d Worker ID: adf20b27088c9ea2dd7ff68f82bcb8fe0b6ecc05e7733c41c26786bc Node ID: 0404e9179a476296bf9e0705bbbe46126925b8378795355858071fd7 Worker IP address: 10.64.19.132 Worker port: 40249 Worker PID: 885669\n",

``` 



Thank you for all your assistance!

**Comment by leoin841023** [comment]
Or maybe you can tell me that 
how to use these models and parameter
``
                                                      auto_stack=True,
                                                        time_limit=100,
                                                        hyperparameters={  
                                                        'NN': {'ag_args_fit': {'num_gpus': 1}},    
                                                        KNNRapidsModel : {},
                                                        LinearRapidsModel : {},
                                                        RFRapidsModel : {'n_estimators': 100},
                                                        'XGB': {'ag_args_fit': {'num_gpus': 1}},
                                                        'FASTAI': {'ag_args_fit': {'num_gpus': 1}},
``
Which version should I install, and what other dependency packages?
